presentations/en/Strategy-Continuous-Learning-Automating-Improving.md

```diff
@@ -0,0 +1,141 @@
+# IT Strategy for IITB
+
+## Continuous Learning, Automation and Improving
+
+### 20% Time
+
+FY 2019-2020 Q4
+
+---
+
+## Goals
+
+- Continuous learning, automation and improvement
+- Agile (break down work, sprints, retrospectives)
+- Work in the open (data, information and code)
+- Expose internal and external (all) services as APIs
+- Teams interact through automated digital services
+- Reduce lead time for changes and delivery of services
+- Recover faster incidents and errors
+- Reduce frenquency of incidents and errors
+- Show improvement
+
+--
+
+### Align with GC IT Direction
+
+- [Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html)
+- [Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html)
+- [Policy](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) and [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+- [Mandatory Procedures for EA Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602)
+- [Mandatory Procedures on APIs](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604)
+- [Acceptable Network and Device Use](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32605)
+
+---
+
+## Effective Immediatly - IT teams must
+
+- Use 10% of their time to learn
+- Use 10% of their time to automate and improve
+- Work using Agile methodologies
+- Work in the open
+- Have access to resources and modern IT tools
+
+---
+
+### IT teams must use 10% of their time to learn
+
+- Try new technologies
+- Attend events and conferences
+- Stay up to date on trends and technologies
+- Share knowledge (IITB showcase, Dev CoP, ..)
+
+> Make it a requirement in performance agreements of directors, managers and team leads
+
+---
+
+### IT teams must use 10% of their time to automate and improve existing processes
```

Comment:
IT teams must use 10% of their time to automate and improve existing processes
=> 
IT teams must use 10% of their time to automate or improve existing processes

Some teams may not have expertise to automate ... 

Which, then again, in this case they should spend more time learning ... so should be really be diving these two? 10 and 10?

Just 20% and let the teams self manage how they use it would be preferable, no?